.circleci/config.yml

```diff
@@ -157,7 +157,7 @@ jobs:
     steps: # a collection of executable commands
       - checkout
       - restore_cache:
-          key: dockstore-web-cache-unit-test-{{ .Environment.CACHE_VERSION }}
+          key: dockstore-web-cache-unit-test-{{ .Environment.CACHE_SEGMENTED_VERSION }}-segmented
```

Comment:
Why did this need to change? Seems fine, but I'm not clear why `CACHE_VERSION` didn't already work.

Reply:
`dockstore-web-cache-unit-test-{{ .Environment.CACHE_VERSION }}-segmented` could probably work too
Feels a little weird that if someone bumped the variable, both types of cache would iterate. 
But after I actually squash this into develop, its probably a little moot

Reply:
Ah, I see, because it will conflict with what's on develop. Makes sense.